### DISCUSSION TITLE
Issue with the CORS middleware documentation

### DISCUSSION THREAD

**Discussion by dfioravanti** [encode/starlette#2206] [Potential Issue] [answered]
Hey everyone, thanks a lot for this amazing project. I just noticed that the documentation for the [CORS middleware](https://www.starlette.io/middleware/) does not mention that `allow_headers=True` cannot work with `allow_origins=["*"]`. For example this is  specified in the fastapi documentation about [CORS](https://fastapi.tiangolo.com/tutorial/cors/). Looking at the [mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#credentialed_requests_and_wildcards), it looks like fastapi is right. Is possible to fix it? Or should I open a bug request?

**Comment by alex-oleshkevich** [comment, marked as answer]
You can file an issue OR send a PR. That would be a nice contribution!
